config/releases.exs

```diff
@@ -73,8 +78,6 @@ if Environment.exists?("BASIC_AUTH_USERNAME") do
     ]
 end
 
-config :elixir_boilerplate, :corsica, origins: Environment.get_list("CORS_ALLOWED_ORIGINS")
```

Comment:
Same here? 🤔

Reply:
Same answer here, it make more sense to me to define the Corsica configuration right after Endpoint, before an optional blok to enable basic auth.